YAXLib/YAXExceptions.cs

```diff
@@ -549,15 +690,15 @@ public class YAXBadlyFormedXML : YAXException
         /// Initializes a new instance of the <see cref="YAXBadlyFormedXML"/> class.
         /// </summary>
         /// <param name="innerException">The inner exception.</param>
-        public YAXBadlyFormedXML(Exception innerException)

```

Comment:
This is also a deserialization exception. Shouldn't it also inherit from DeserializationException?